bundles/org.openhab.binding.matter/src/main/java/org/openhab/binding/matter/internal/controller/devices/converter/ColorControlConverter.java

```diff
@@ -0,0 +1,483 @@
+/*
+ * Copyright (c) 2010-2025 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.matter.internal.controller.devices.converter;
+
+import static org.openhab.binding.matter.internal.MatterBindingConstants.CHANNEL_COLOR_COLOR;
+import static org.openhab.binding.matter.internal.MatterBindingConstants.CHANNEL_COLOR_TEMPERATURE;
+import static org.openhab.binding.matter.internal.MatterBindingConstants.CHANNEL_COLOR_TEMPERATURE_ABS;
+import static org.openhab.binding.matter.internal.MatterBindingConstants.CHANNEL_ID_COLOR_COLOR;
+import static org.openhab.binding.matter.internal.MatterBindingConstants.CHANNEL_ID_COLOR_TEMPERATURE;
+import static org.openhab.binding.matter.internal.MatterBindingConstants.CHANNEL_ID_COLOR_TEMPERATURE_ABS;
```

Comment:
For static imports, this can be shortened (and is allowed):
```suggestion
import static org.openhab.binding.matter.internal.MatterBindingConstants.*;
```